docs/syntax_guide.rst

```diff
@@ -136,27 +141,33 @@ Properties added by `OrientedPoint`:
 ===================  ==============  ================================================
    **Property**       **Default**                    **Meaning**
 -------------------  --------------  ------------------------------------------------
- heading [1]_          0             heading in global coordinates
- viewAngle            360 degrees    angle for the ‘can see’ operator
- headingStdDev         5 degrees     mutation standard deviation for :prop:`heading`
+ yaw [1]_             0              yaw in local coordinates
+ pitch [1]_           0              pitch in local coordinates
+ roll [1]_            0              roll in local coordinates
+ parentOrientation    global         basis for local coordinate system
+ viewAngles           (2π, π)        angles for visibility calculations
+ headingStdDev        5 degrees      mutation standard deviation for :prop:`heading`
```

Comment:
This no longer exists. (Maybe we should have `orientationStdDev` depend on it for backwards compatibility? Otherwise we'll have to make a note in the list of incompatible changes. In any case `orientationStdDev` needs to be added to the table and we should remove `headingStdDev`.)

Reply:
Good idea, I'll make the change to add the dependency.